### ISSUE TITLE
find() and findAll() cannot select one class tag out of multiple

### ISSUE THREAD

**Issue by RLesser** [chishui/JSSoup#7]
For instance, if this were my html structure in variable `soup`:
```html
<html>
  <body>
    <div class="entry top-entry"> entry one </div>
    <div class="entry"> entry two </div>
  </body>
</html>
```
Then it is not currently possible to retrieve all divs with the entry class.
Searching for entry retrieves only the second element:
```javascript
soup.findAll("div", "entry").length
\\ 1
soup.findAll("div", "entry")[0].text
\\ 'entry two'
```
In order to get the first element, one must search for the complete class entry:
```javascript
soup.find("div", "entry top-entry").text
\\ 'entry one'
```

The whole point of class tags is that you can assign multiple to an element and select by them individually, so this is currently a pretty drastic flaw in the code.

**Comment by chishui** [comment]
Hi, I believe this issue has been fixed, I just publish version 0.0.7. Please update to this version and test again. Thanks

**Comment by kevin6shah** [comment]
I am having a similar issue but it’s the case where it can’t find the element with class = “entry top-entry” (classes with spaces) for some websites like (H&M) but works for others. I have also checked that all my html is extracted correctly before sending it to JSSoup.
